encoding/ssz/detect/configfork_test.go

```diff
@@ -297,7 +297,7 @@ func TestDetectAndUnmarshalBlock(t *testing.T) {
 			name:      "genesis block in altair slot",
 			b:         signedTestBlockGenesis,
 			slot:      bellaS - 1,
-			errExists: true,
+			errExists: false,
```

Comment:
What's the reason for this change?

Reply:
As the block structrure of altair is same as phase 0, genesis block can be processed in altair slot